projects/plugins/jetpack/tests/php/core-api/wpcom-endpoints/test-class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -252,6 +252,44 @@ public function submenu_item_data() {
 		);
 	}
 
+	/**
+	 * Check if the menu URL is properly generated from the first submenu slug.
+	 */
+	public function test_if_the_first_submenu_url_is_used_for_menu_url() {
+		global $menu;
+
+		add_menu_page( '', __( 'Foo', 'jetpack' ), 'read', 'foo' );
```

Comment:
Nitpick: Given this is a test, we don't need to translate the title.
```suggestion
		add_menu_page( '', 'Foo', 'read', 'foo' );
```